R/tm_g_association.R

```diff
@@ -330,9 +330,13 @@ srv_tm_g_association <- function(id,
       selector_list = selector_list
     )
 
+    qenv <- teal.code::eval_code(
+      data(),
+      'library("ggplot2");library("dplyr")'
```

Comment:
This plot is based on `tm_g_bivariate` and use functions from another file
```suggestion
      'library("ggplot2");library("dplyr");library("tern");library("ggmosaic");library("grid")'
```

<img width="736" alt="image" src="https://github.com/user-attachments/assets/f314664f-7861-4a17-a014-f3f59b6cfe3e" />

Reply:
Good catch, I think I thought that prefixing would be enough on this case. But I also see an aes without `ggplot2::`